src/measure_theory/measure/finite_measure_weak_convergence.lean

```diff
@@ -737,6 +770,186 @@ end probability_measure -- namespace
 
 end probability_measure -- section
 
+section normalize_finite_measure
+/-! ### Normalization of finite measures to probability measures
+
+This section is about normalizing finite measures to probability measures.
+
+The weak convergence of finite measures to nonzero limit measures is characterized by
+the convergence of the total mass and the convergence of the normalized probability
+measures.
```

Comment:
This is true and important, but I can't see the lemmas stating explicitly this below.

Also, are you aware of the file `measure_theory/integral/average.lean`? It could be worth setting up a small API to link the two (for instance saying that the average is the integral with respect to the normalized measure).

Reply:
I did not know about `average`. I added the lemma `finite_measure.average_eq_integral_normalize`. Maybe averages over sets and normalized restrictions of finite measures could be meaningful, but I would not include in this PR.

Reply:
Thanks for reminding about the actual statements about characterizing weak convergence! I had written only the convergence of the associated integrals of test functions, which is the main characterization of weak convergence, but not the actual weak convergence. I now added three lemmas that actually talk about weak convergence: `tendsto_normalize_iff_tendsto` (which is the "iff" characterization, but has an annoying nonzero assumption) and the two directions `tendsto_normalize_of_tendsto` and `tendsto_of_tendsto_normalize_test_against_nn_of_tendsto_mass` (the latter of which can avoid nonzero assumptions).